### PR TITLE
Bug tag name and abilities 2

### LINKED ISSUE #83: Incorrect ability tags
Similar to #23, I figured having a spot to mention inconsistencies in the ability search. So far I have found:
Excalibur Batman incorrectly showing Drone Mazes, Electricity, Flying, Hazard Protection, Intelligence, Lantern Constructs, Laser, Magic, Raven Portals
Excalibur Batman not showing Boomerang, Cracked LEGO/Super Strength (these should be merged into one ability), Grapple, Laser Deflector, Master Build, Stealth, Sword Switch
Legolas not showing Grind Rails (not that it matters, just worth mentioning)
Merge Growth and Water Spray into one thing, they both do the same thing
No P.K.E. Meter ability
No Rope Swings ability (This is different than Grapple, only Year 2 grapple characters can do it)
Add Technology, Silver LEGO Blowup, and Grapple to Abby Yates for Erin and Jillian (other abilities that Abby herself doesn't have seem to be accounted for)
Add Mini Access, Intelligence, Silver LEGO Blowup, Acrobat, Laser Deflector, Technology, X-Ray Vision, Fix-It, Tracking, Illumination, Sonar Smash, and Vine Cut to Sloth for the rest of the Goonies. 
If I notice anything else, I'll mention it

### PR DESCRIPTION
**Pull request by VladimirKuletski**
Another set of edits for abilities description.
https://lego-dimensions.fandom.com/wiki/List_of_Playable_Characters used as a base.

1. Merged all skills from other forms into Abby
2. Reviewed all characters with "Cracked LEGO Objects" and move it to "Super Strength" together with skills update
3. No "Melt ice" as it's included into "Laser"
4. Added "Character Changing" as it was mentioned in skills on website above
5. Converted all variants to "Hearts Regenerate"

Partial fix #83 

@Luigimeansme please also review

### CODE REVIEW

**Review by Berny23 — APPROVED** [review]

**Comment by Luigimeansme** [comment]
@Berny23 Are we waiting on something specifically before this gets merged?

**Pull request merged**
